src/src/rx_main.cpp

```diff
@@ -825,7 +825,7 @@ void loop()
         RFmodeLastCycled = millis();
     }
 
-    if ((connectionState == connected) && ((millis() > (LastValidPacket + ExpressLRS_currAirRate_RFperfParams->RFmodeCycleInterval)) || ((millis() > (LastSyncPacket + 11000)) && uplinkLQ < 10))) // check if we lost conn.
+    if ((connectionState == connected) && (ExpressLRS_currAirRate_RFperfParams->RFmodeCycleInterval < (millis() - LastValidPacket))) // check if we lost conn.
```

Comment:
Cast to `int32_t` to avoid possible race condition:
`(int32_t)(millis() - LastValidPacket)`